example-chat-task/conf.yml

```diff
@@ -64,6 +64,8 @@ _mturk:
     LifetimeInSeconds: 1209600              # 14 days = 1.2M sec
     AssignmentDurationInSeconds: 3600      # 1 hour = 3.6k sec
     Reward: *reward
+    BonusRate: 15.00                       # Dollars per hour
```

Comment:
this should be named something like DesiredRate with a comment that it's the desired hourly rate that the bonus is meant to allow the user to reach.